packages/snaps-controllers/src/services/AbstractExecutionService.ts

```diff
@@ -64,6 +64,10 @@ export type TerminateJobArgs<WorkerType> = Partial<Job<WorkerType>> &
 export abstract class AbstractExecutionService<WorkerType>
   implements ExecutionService
 {
+  name = 'ExecutionService' as const;
```

Comment:
```suggestion
  name = constrollerName as const;
```

Reply:
> TS1355: A const assertions can only be applied to references to enum members, or string, number, boolean, array, or object literals.

Changed it to `name: typeof controllerName = controllerName;`

Reply:
I edited my suggestion as I realized that too 😄